### PR TITLE
Add Portfolio Listing

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Adds Portfolio Listing module to Companies page.
- - Adds missing keys to `InvestmentsList`

### Type(s) of changes

- [x] New feature

### How Has This Been Tested?
View with FP link.
@featurepeek /companies

### Applicable screenshots:

image
image

### Follow-up PR

- Design may change during QA re: the grid @mokaymokay ! I'd like to have it ready for them to see during QA and make changes if needed.

### CODE REVIEW

File: src/components/layouts/PortfolioListing.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <div
+      className={cx('PortfolioListing site-max-width site-padding-x md:pt3_75 mxauto', className)}
+    >
+      {sectors.map((sector: Sector) => (
+        <div
+          key={`PortfolioListing--${sector.name}`}
+          className="PortfolioListing__sectors-container flex flex-row flex-wrap"
+        >
+          {sector.companies &&
+            sector.companies.map((company: Company) => (
```
Comment (mokaymokay) [inline src/components/layouts/PortfolioListing.tsx:26]: might not need the check on line 25, if the array is empty, map will just return null

File: src/components/layouts/PortfolioListing.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+export default PortfolioListing;
+
+const FounderCard = (founder: Founder) => {
+  return (
+    <div
+      key={`PortfolioListing-FounderCard-${generateFullName(founder)}`}
+      className="FounderCard relative"
+    >
+      <div className="z-3 absolute t0 r0 l0 w100 h100 flex flex-col items-start">
+        <p className="absolute pb_75 pl_75 b0 l0 uppercase color-lilac primary-sm">
```
Comment (mokaymokay) [inline src/components/layouts/PortfolioListing.tsx:67]: might not need `absolute`?

File: src/styles/components/PortfolioListing.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
+.PortfolioListing {
+  padding-top: 8.625rem;
+
+  &__company-container {
+    &:hover {
```
Comment (mokaymokay) [inline src/styles/components/PortfolioListing.scss:5]: might want to show this effect on focus too?

File: src/styles/components/PortfolioListing.scss [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        padding: 1.1875rem 0.625rem;
+      }
+    }
+  }
+
+  .FounderCard {
+    width: 50%;
+
+    @include media('md-up') {
+      width: unset;
```
Comment (mokaymokay) [inline src/styles/components/PortfolioListing.scss:49]: doesn't seem like you need the `unset`

**Review by mokaymokay — APPROVED** [review]

File: src/components/layouts/PortfolioListing.tsx (hunk 2 above)
Comment (chrismekim) [reply src/components/layouts/PortfolioListing.tsx:67]: Thanks for reviewing so quickly before I'm OOO!!! 

Hmm tried it without and seems I need absolute here?

**Pull request merged**